modules/core/src/test/java/org/apache/ignite/internal/processors/cache/mvcc/CacheMvccAbstractTest.java

```diff
@@ -1716,12 +1716,19 @@ protected MvccProcessorImpl mvccProcessor(Ignite node) {
      * @throws Exception If failed.
      */
     protected final void checkActiveQueriesCleanup(Ignite node) throws Exception {
-        final MvccProcessorImpl crd = mvccProcessor(node);
+        final MvccProcessorImpl prc = mvccProcessor(node);
```

Comment:
Let's rename to 'proc'

Reply:
Not sure if it's really needed here. There are too many places where this name is used. We may easily rename all of them but in separate commit.